misc/userscripts/qute-pass

```diff
@@ -58,6 +60,8 @@ argument_parser.add_argument('url', nargs='?', default=os.getenv('QUTE_URL'))
 argument_parser.add_argument('--password-store', '-p',
                              default=os.getenv('PASSWORD_STORE_DIR', default=os.path.expanduser('~/.password-store')),
                              help='Path to your pass password-store')
+argument_parser.add_argument('--mode', '-M', choices=['pass', 'gopass'], default="pass",
+                             help='Select mode [gopass] to enable gopass-specific features like mounts.')
```

Comment:
The help text is a bit misleading, especially when the user only has `gopass` installed.

```suggestion
                             help='Select mode [gopass] to use gopass instead of the standard pass.')
```